mne/minimum_norm/tests/test_inverse.py

```diff
@@ -44,6 +45,12 @@
 fname_label = op.join(s_path, 'labels', '%s.label')
 fname_vol_inv = op.join(s_path,
                         'sample_audvis_trunc-meg-vol-7-meg-inv.fif')
+# trans and bem needed for channel reordering tests incl. forward computation
+fname_trans = op.join(s_path, 'sample_audvis_trunc-trans.fif')
+s_path_bem = op.join(testing.data_path(download=False), 'subjects',
```

Comment:
More DRY here to use `s_path` here instead of re-calling `testing.data_path` again